gr-digital/lib/protocol_formatter_bb_impl.cc

```diff
@@ -67,7 +67,7 @@ namespace gr {
     }
 
     int
-    protocol_formatter_bb_impl::work(int noutput_items,
+    protocol_formatter_bb_impl::work(size_t noutput_items,
```

Comment:
We have to change almost every block if we change this. In total, this seems more work than just doing `static_cast<>()` whenever we get a signed/unsigned warning.